docs/Tutorials/Private-Network/Create-QBFT-Network.md

```diff
@@ -378,6 +378,9 @@ When finished using the private network, stop all nodes using ++ctrl+c++ in each
 
 <!-- Links -->
 [private network example tutorial]: ../Examples/Private-Network-Example.md#creating-a-transaction-using-metamask
-
+[block header validator selection method]: ../../HowTo/Configure/Consensus-Protocols/Add-Validators.md#adding-and-removing-validators-by-voting
```

Comment:
```suggestion
[block header validator selection method]: ../../HowTo/Configure/Consensus-Protocols/Add-Validators.md#qbft
```

There are two headers named "Adding and removing validators by voting" on this page, so linking it actually links the first instance (under IBFT 2.0, not QBFT). I suppose next best is linking to the QBFT section.

Reply:
Good catch! If multiple topics headings exist with the same name then you can append "_<num>", where num is the instance of the heading to link to. Here I'll link to "#adding-and-removing-validators-by-voting_1" (the second instance)